twitter-export-image-fill.py

```diff
@@ -58,6 +58,24 @@
 
 image_count_global = 0
 
+# Function to download an avatar (from tweet or retweet)
+
+def download_avatar(user):
```

Comment:
Ok will do. It basically expects a dictionary that corresponds to the user stanza of a tweet or retweet.